docs/pages/Kratos/Expressions/General/Expression_Types.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Expression Types
+keywords: 
+tags: [Expression_Types.md]
+sidebar: kratos_expressions
+summary: 
+---
+
+## Introduction
+
+There are three types of expressions.
+* NodalExpression
+* ConditionExpression
+* ElementExpression
+
+## Nodal Expression
+
+Nodal expressions are used to store data related to nodal quantities. They may be historical or non-historical. Following code snippet illustrtes creating a nodal expression
+and reading ```VELOCITY``` from nodal non-historical data value container.
```

Comment:
Even though the name of the container class is indeed `DataValueContainer`, it's a stupid name that need not be exposed to the reader.


```suggestion
and reading ```VELOCITY``` from non-historical nodal container.
```